alma/alma-8.x/common/install_docker.sh

```diff
@@ -28,34 +24,32 @@ yum install -y nvidia-container-runtime
 # Mark the installed packages on hold to disable updates
 sed -i "$ s/$/ *nvidia-container*/" /etc/dnf/dnf.conf
 
-wget https://raw.githubusercontent.com/NVIDIA/nvidia-docker/master/nvidia-docker
-cp nvidia-docker /bin/
-chmod +x /bin/nvidia-docker
-wget https://raw.githubusercontent.com/NVIDIA/nvidia-docker/master/daemon.json
-cp daemon.json /etc/docker/
+nvidia-ctk runtime configure --runtime=docker
```

Comment:
why are we changing this?

Reply:
> why are we changing this?

Because nvidia changed how the container toolkit works and it doesn't install their own docker anymore. The installation instructions are updated on the website.